editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -1514,6 +1514,7 @@ VisualShaderEditor::VisualShaderEditor() {
 	add_options.push_back(AddOption("CubeMapUniform", "Textures", "Variables", "VisualShaderNodeCubeMapUniform", TTR("Cubic texture uniform."), -1, VisualShaderNode::PORT_TYPE_COLOR));
 	add_options.push_back(AddOption("TextureUniform", "Textures", "Variables", "VisualShaderNodeTextureUniform", TTR("2D texture uniform."), -1, VisualShaderNode::PORT_TYPE_COLOR));
 
+	add_options.push_back(AddOption("Flipbook", "Textures", "Functions", "VisualShaderNodeFlipbook", TTR("Flipbook animation the texture, eg min 0.1 = 1 to max 1.0 = 10 (amount textures)"), -1, VisualShaderNode::PORT_TYPE_VECTOR));
```

Comment:
"Flipbook animation the texture, eg min 0.1 = 1 to max 1.0 = 10 (amount textures)" is this correct or a little change?

Reply:
as I said - 1 - mean 1 frame, 5 means - 5 frames and so on

Reply:
The maximum number of frames should be configurable in a wide range, it's better to use a cycle to generate a code in that case.

Reply:
strange but the user will be incomprehensible
1 frame = min  0.1 amount texture
10 frames = max 1.0 we do not have an integer so  only float

Reply:
Then convert it to that range - thus the 10 frames are pretty much useless to show a complex animation(and as @mrcdk mentioned below it's better to use AnimatedTexture for that), if you really need this node there should be the configurable range - that's my opinion

Reply:
![image](https://user-images.githubusercontent.com/17591539/56202205-2faa1480-604b-11e9-9061-c05a4f588f39.png)

I understand the maximum 10 frames a little but the visual shader is not enough
add enter port +/- extension frame it's not implemented

Reply:
Well, I think it's possible - if this is not possible then it's a subject for implement, prior this PR

Reply:
@mrcdk Can you do `AnimatedTexture` for a titlemap? it's not, only shader or visual shader

Reply:
Yes, it can be used anywhere a `Texture` can be used.

Reply:
okay merge pull or refuse, here discussion no longer makes sense (i don't care)